relay-profiling/src/android.rs

```diff
@@ -221,6 +238,17 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_roundtrip_react_native() {
```

Comment:
This test validates the parsing doesn't fail, but we don't know if the contents are what we expect. What do you think about validating that too?

Reply:
Test ensuring values are in current places are in `sample.rs`.